### DISCUSSION TITLE
[Unity] 예진의 Unity UI RectTransform 개념

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#114] [1. 개발 관련 글 모음]
일하면서 뭔가 정리가 안 돼서 끄적였는데 예제가 좀 마음에 안 드네요 ㅠㅠ
내일 스크립트랑 예제 수정 해봐야겠어요..!
[블로그 링크](https://donkeysdevelpment.tistory.com/127)

**Comment by shkisme** [comment]
헉 볼려고 했는 데 권한이 없는 페이지래요...!

**Reply by pnudpwls1103** [reply]
비공개로 올라가버렸네요 ㅋㅋㅋ
알려주셔서 감사합니다 홀홀홀
